lms/djangoapps/teams/static/teams/js/views/topic_card.js

```diff
@@ -1,52 +1,58 @@
                 className: 'team-count',
 
-                initialize: function () {
+                initialize: function() {
                     this.render();
                 },
 
-                render: function () {
-                    var team_count = this.model.get('team_count');
-                    this.$el.html(_.escape(interpolate(
-                        ngettext('%(team_count)s Team', '%(team_count)s Teams', team_count),
-                        {team_count: team_count},
-                        true
-                    )));
+                render: function() {
+                    var teamCount = this.model.get('team_count');
+                    this.$el.html(HtmlUtils.interpolateHtml(
+                        ngettext('{team_count} Team', '{team_count} Teams', teamCount),
+                        {team_count: teamCount}
+                    ).toString());
                     return this;
                 }
             });
 
             var TopicCardView = CardView.extend({
-                initialize: function () {
-                    this.detailViews = [new TeamCountDetailView({ model: this.model })];
+                initialize: function() {
+                    this.detailViews = [new TeamCountDetailView({model: this.model})];
                     CardView.prototype.initialize.apply(this, arguments);
                 },
 
-                actionUrl: function () {
+                actionUrl: function() {
                     return '#topics/' + this.model.get('id');
                 },
 
                 configuration: 'square_card',
                 cardClass: 'topic-card',
                 pennant: gettext('Topic'),
-                title: function () { return this.model.get('name'); },
-                description: function () { return this.model.get('description'); },
-                details: function () { return this.detailViews; },
+                title: function() { return this.model.get('name'); },
+                description: function() { return this.model.get('description'); },
+                details: function() { return this.detailViews; },
                 actionClass: 'action-view',
-                actionContent: function () {
-                    var screenReaderText = _.escape(interpolate(
-                        gettext('View Teams in the %(topic_name)s Topic'),
-                        { topic_name: this.model.get('name') }, true
-                    ));
-                    return '<span class="sr">' + screenReaderText + '</span><i class="icon fa fa-arrow-right" aria-hidden="true"></i>';
+                actionContent: function() {
+                    var screenReaderText = HtmlUtils.interpolateHtml(
+                        gettext('View Teams in the {topic_name} Topic'),
+                        {topic_name: this.model.get('name')}
+                    );
+                    return HtmlUtils.HTML(
+                        '<span class="sr">' +
+                        screenReaderText +
```

Comment:
This looks alarming.  Some options include:
1. Rename `screenReaderText` to `screenReaderTextHtml`, or
2. Find if there is a way to get this actual HTML into the existing calling template, or  
3. Make this whole thing an inline template.

If you went with the third option, I'm still not loving that `HtmlUtils.template()` doesn't return an HtmlSnippet.  I guess I just need to forget about it.  Worst case we have double-escaping that someone needs to fix.